lnwire/node_announcement_test.go

```diff
@@ -67,26 +66,3 @@ func TestNodeAnnoucementValidation(t *testing.T) {
 		t.Fatal(err)
 	}
 }
-
-func TestNodeAnnoucementPayloadLength(t *testing.T) {
-	na := &NodeAnnouncement{
-		Signature: someSig,
-		Timestamp: maxUint32,
-		Address:   someAddress,
-		NodeID:    pubKey,
-		RGBColor:  someRGB,
-		pad:       maxUint16,
-		Alias:     someAlias,
-	}
-
-	var b bytes.Buffer
-	if err := na.Encode(&b, 0); err != nil {
-		t.Fatalf("unable to encode node: %v", err)
-	}
-
-	serializedLength := uint32(b.Len())
-	if serializedLength != na.MaxPayloadLength(0) {
-		t.Fatalf("payload length estimate is incorrect: expected %v "+
-			"got %v", serializedLength, na.MaxPayloadLength(0))
-	}
-}
```

Comment:
Maybe rather than removing it, we consider check the exact length of message instead?